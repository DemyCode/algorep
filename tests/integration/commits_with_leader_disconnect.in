1
5
no
timeout 2
sleep 350
entry 1 test
start 1
wait_finish 1
sleep 350
print_local_log 2
print_local_log 3
print_local_log 4
print_local_log 5
print_local_log 6
crash 2
sleep 350
get_state 2
sleep 365
entry 1 test2
wait_finish 1
sleep 250
print_local_log 3
print_local_log 4
print_local_log 5
print_local_log 6
recover 2
sleep 650
get_state 2
entry 1 test3
wait_finish 1
sleep 250
print_local_log 2
print_local_log 3
print_local_log 4
print_local_log 5
print_local_log 6
stop 1
stop 2
stop 3
stop 4
stop 5
stop 6
