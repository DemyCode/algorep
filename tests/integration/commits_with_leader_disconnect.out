2 [1, "test"]
3 [1, "test"]
4 [1, "test"]
5 [1, "test"]
6 [1, "test"]
State 2: STOPPED
3 [1, "test"] [2, "test2"]
4 [1, "test"] [2, "test2"]
5 [1, "test"] [2, "test2"]
6 [1, "test"] [2, "test2"]
State 2: FOLLOWER
2 [1, "test"] [2, "test2"] [2, "test3"]
3 [1, "test"] [2, "test2"] [2, "test3"]
4 [1, "test"] [2, "test2"] [2, "test3"]
5 [1, "test"] [2, "test2"] [2, "test3"]
6 [1, "test"] [2, "test2"] [2, "test3"]
